plugin/core/windows.py

```diff
@@ -312,9 +306,7 @@ def __init__(
         configs: WindowConfigManager,
         documents: DocumentHandler,
         diagnostics: DiagnosticsStorage,
-        session_starter: Callable,
         sublime: Any,
```

Comment:
Can we remove `sublime` from the constructor? :)

Reply:
I would like to, but not for this PR.